packages/augur-contract-dependencies-ethers/src/ContractDependenciesEthers.ts

```diff
@@ -268,18 +272,14 @@ export class ContractDependenciesEthers implements Dependencies<BigNumber> {
     tx: Transaction<ethers.utils.BigNumber>,
     txMetadata: TransactionMetadata
   ): Promise<ethers.providers.TransactionReceipt> {
-    const gasLimit = tx.gasLimit || (await this.provider.estimateGas(tx));
-    const gasPrice = await this.provider.getGasPrice();
-    tx.gasPrice = gasPrice;
+    tx.gasLimit = tx.gasLimit || await this.provider.estimateGas(tx);
+    tx.gasPrice = tx.gasPrice || await this.provider.getGasPrice();
```

Comment:
This behavior is new and might(?) break something: we can now specify gas price per-tx and it will pass through to the signer. The signer can then do whatever it wants of course.

Reply:
Yea this should be reasonable and if not something else was wrong ;p